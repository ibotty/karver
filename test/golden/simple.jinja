textVar: {{textVar}}.
listVar: {% for elem in listVar %} {{ elem }} {% endfor %}
