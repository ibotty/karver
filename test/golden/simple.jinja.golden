textVar: some text.
listVar:  a  list 